tsdb/index/tsi1/index.go

```diff
@@ -876,7 +876,6 @@ func (i *Index) compactGroup(files []*IndexFile) {
 		log.Printf("%s: error creating compaction files: %s", IndexName, err)
 		return
 	}
-	defer f.Close()
```

Comment:
@oiooj Can you show the panic that's occurring? Closing twice shouldn't be a problem.